src/org/jitsi/util/RTPUtils.java

```diff
@@ -217,4 +217,29 @@ else if (a > b)
         }
     };
 
+    /**
+     * Returns the difference between two RTP timestamps.
+     * @return the difference between two RTP timestamps.
+     */
+    public static long rtpTimestampDiff(long a, long b)
+    {
+        long diff = a - b;
+        if (diff < -(1L<<31))
+            diff+= 1L<<32;
+        else if (diff > 1L<<31)
+            diff-= 1L<<32;
+
+        return diff;
```

Comment:
actually i meant to mark it as deprecated but i was worried about removing it completely in case it's being used somewhere.  is it alright to just remove it?